app/routes/surveys/components/AdminSideBar.tsx

```diff
@@ -69,9 +72,8 @@ const AdminSideBar = ({
               Rediger
             </LinkButton>
 
-            {actionGrant &&
-              actionGrant.includes('csv') &&
-              exportSurvey &&
+            {actionGrant.includes('csv') &&
```

Comment:
`actionGrant` might be `undefined`, no?
```suggestion
            {actionGrant?.includes('csv') &&
```